catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java

```diff
@@ -204,12 +204,11 @@ public JdbcSchema loadSchema(NameIdentifier ident) throws NoSuchSchemaException
     }
     Map<String, String> properties =
         load.properties() == null ? Maps.newHashMap() : Maps.newHashMap(load.properties());
-    StringIdentifier.newPropertiesWithId(id, properties);
     return new JdbcSchema.Builder()
         .withAuditInfo(load.auditInfo())
         .withName(load.name())
         .withComment(StringIdentifier.removeIdFromComment(load.comment()))
-        .withProperties(properties)
+        .withProperties(StringIdentifier.newPropertiesWithId(id, properties))
```

Comment:
Can you show me the logic of why `stringId` will impact the the audit info?

Reply:
gravitino Id is used to retrieve audit info from the entity store, if missing, it will use the audit load from the Jdbc catalog which is  empty.